cmd/internal/planetscale_edge_database.go

```diff
@@ -171,7 +171,9 @@ func (p PlanetScaleEdgeDatabase) Read(ctx context.Context, w io.Writer, ps Plane
 	)
 
 	tabletType := psdbconnect.TabletType_primary
-	if ps.UseReplica {
+	if ps.UseRdonly {
+		tabletType = psdbconnect.TabletType_batch
```

Comment:
Confirmed that this will be mapped to `topodatapb.TabletType_RDONLY` by our edge gateway.